test/ffmpeg-qsv/encode/hevc.py

```diff
@@ -363,3 +363,46 @@ def test(self, case, gop, bframes, bitrate, maxrate, profile, rcmode):
     self.init(spec, case, gop, bframes, bitrate, maxrate, profile, rcmode)
     self.encode()
 
+class seek(HEVC8EncoderTest):
+  def init(self, tspec, case, gop, bframes, bitrate, qp, maxrate, profile, rcmode, seek):
+    vars(self).update(tspec[case].copy())
+    vars(self).update(
+      rcmode     = rcmode,
+      bframes    = bframes,
+      bitrate    = bitrate,
+      case       = case,
+      maxrate    = maxrate,
+      minrate    = bitrate,
+      profile    = profile,
+      qp         = qp,
+      gop        = gop,
+      vseek      = 1,
```

Comment:
what is "vseek" used for?

Reply:
Sorry I ignored this, it's been updated.